packages/mdc-tabs/tab/mdc-tab.scss

```diff
@@ -0,0 +1,177 @@
+//
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+@import "@material/animation/functions";
+@import "@material/typography/mixins";
+@import "@material/theme/mixins";
+@import "@material/ripple/mixins";
+@import "@material/rtl/mixins";
+
+$mdc-tab-with-text-height: 48px;
+$mdc-tab-with-icon-and-text-height: 72px;
+
+@mixin mdc-tabs-icon-tab-content {
+  display: block;
+  margin: 0 auto;
+}
+
+// postcss-bem-linter: define tab
+.mdc-tab {
+  @include mdc-typography(body2);
+
+  position: relative;
+  box-sizing: border-box;
+  display: table-cell;
+  text-align: center;
+  vertical-align: middle;
+  min-height: $mdc-tab-with-text-height;
+  min-width: 160px;
+  overflow: hidden;
+  padding: 0 24px;
+  text-decoration: none;
+  cursor: pointer;
+  white-space: nowrap;
+
+  @include mdc-theme-prop(color, text-secondary-on-light);
+
+  &:hover {
+    @include mdc-theme-prop(color, text-primary-on-light);
+  }
+
+  &:focus {
+    @include mdc-theme-prop(outline-color, text-secondary-on-light);
+  }
+
+  @include mdc-theme-dark(".mdc-tab-bar") {
+    @include mdc-theme-prop(color, text-secondary-on-dark);
+
+    &:hover {
+      @include mdc-theme-prop(color, text-primary-on-dark);
+    }
+
+    &:focus {
+      @include mdc-theme-prop(outline-color, text-secondary-on-dark);
+    }
+  }
+
+  // TODO: Replace with breakpoint variable
+  @media screen and (max-width: 600px) {
+    min-width: 72px;
+    padding: 0 12px;
+  }
+
+  // postcss-bem-linter: ignore
+  &__icon {
+    @include mdc-tabs-icon-tab-content;
+
+    width: 24px;
+    height: 24px;
+
+    // postcss-bem-linter: ignore
+    .mdc-tab-bar--icons-with-text & {
+      margin-top: 4px;
+    }
+  }
+
+  &__icon-text {
+    @include mdc-tabs-icon-tab-content;
+  }
+
+  // postcss-bem-linter: ignore
+  &__icon + &__icon-text {
+    padding-top: 6px;
+  }
+}
+
+.mdc-tab--active {
+  @include mdc-theme-prop(color, text-primary-on-light);
+
+  @include mdc-theme-dark(".mdc-tab-bar") {
+    @include mdc-theme-prop(color, text-primary-on-dark);
+  }
+
+  &:before {
```

Comment:
The way IE11 lays out absolutely positioned `table-cell` elements. This was the only way I could get the indicator to line up correctly. 😢